bucoffea/vbfhinv/vbfhinvProcessor.py

```diff
@@ -335,14 +401,31 @@ def process(self, df):
         both_jets_in_hf = (diak4.i0.abseta > 3.0) & (diak4.i1.abseta > 3.0)
         selection.add('veto_hfhf', ~both_jets_in_hf.any())
 
+        # Leading jet |eta| < 2.9
+        leadak4_not_in_hf = (diak4.i0.abseta < 2.9).any()
+        selection.add('leadak4_not_in_hf', leadak4_not_in_hf)
+
+        # Reject events where the leading jet has momentum > 6.5 TeV
+        leadak4_clean = diak4.i0.pt * np.cosh(diak4.i0.eta) < 6500
+        selection.add('leadak4_clean', leadak4_clean.any())
+
         # Divide into three categories for trigger study
         if cfg.RUN.TRIGGER_STUDY:
-            two_central_jets = (np.abs(diak4.i0.eta) <= 2.4) & (np.abs(diak4.i1.eta) <= 2.4)
-            two_forward_jets = (np.abs(diak4.i0.eta) > 2.4) & (np.abs(diak4.i1.eta) > 2.4)
-            one_jet_forward_one_jet_central = (~two_central_jets) & (~two_forward_jets)
+            two_central_jets = (np.abs(diak4.i0.eta) <= 2.5) & (np.abs(diak4.i1.eta) <= 2.5)
+            two_hf_jets = (np.abs(diak4.i0.eta) > 3.0) & (np.abs(diak4.i1.eta) > 3.0)
+            one_jet_forward_one_jet_central = (~two_central_jets) & (~two_hf_jets)
+
             selection.add('two_central_jets', two_central_jets.any())
-            selection.add('two_forward_jets', two_forward_jets.any())
             selection.add('one_jet_forward_one_jet_central', one_jet_forward_one_jet_central.any())
+        
+        # Mask for 1/5th unlbinding
+        one_fifth_mask = ~pass_all
+        # Only pick each 5 entry in data
+        one_fifth_mask[::5] = True
```

Comment:
Just for future reference: This is not entirely replciable because it will depend on the order of the events in the file. Genereally safer, to use the event number: `mask = (df['event']%5)==0`.

Reply:
Thanks, done in 517c5e45d3c18b78f1176a8a20b4995ea9e66567